lib/novu/client.rb

```diff
@@ -18,6 +18,7 @@
 require "novu/api/subscribers"
 require "novu/api/tenants"
 require "novu/api/topics"
+require "novu/api/workflow_Overrides"
```

Comment:
Shouldn't this be `workflow_overrides` instead of `workflow_Overrides`

Reply:
Correction has been made